apps/analysis/serializers.py

```diff
@@ -420,10 +420,10 @@ class Meta:
             'id',
             'title',
             'team_lead',
-            'project',
             'start_date',
             'end_date',
             'cloned_from',
```

Comment:
We should remove `cloned_from`.

We use serializer in GraphQL for mutations only, since we don't use `cloned_from` as a user input, we can remove from here.